site/content/en/docs/tasks/manage/setup_job_admission_policy.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Setup a Job Admission Policy"
+date: 2024-06-28
+weight: 10
+description: >
+  Implementing Validating Admission Policy to prevent job creation without queue name.
+---
+
+This page shows you how to set up a [Validating Admission Policy](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy) to prevent the creation of jobs without a queue name within a namespace.
+
+## Before you begin
+
+Ensure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool can communicate with your cluster.
+
+## Creating a Validating Admission Policy
+
+To verify a job contains a queue name, check that the `kueue.x-k8s.io/queue-name` label exists and has a value set using the [Common Expression Language (CEL)](https://github.com/google/cel-spec).
```

Comment:
The example below shows you how to set up the Job Admission Policy to reject early all Job or JobSets
without the queue-name if sent to the user namespaces. At the same time, it allows you to create and
execute all Jobs in the namespace dedicated to administrative tasks.

In this example we assume that `manageJobsWithoutQueueName` is disabled, letting the admin
to execute Jobs in the `admin-namespace`. Jobs sent to this namespace aren't rejected, or managed
by Kueue.